core/src/mindustry/ui/dialogs/CustomGameDialog.java

```diff
@@ -25,7 +25,7 @@ public CustomGameDialog(){
 
     void setup(){
         clearChildren();
-        add(titleTable);
+        add(titleTable).growX().row();
```

Comment:
You add a `.row()`, which is like a newline in tablespeak, yet directly under this is another `row()`. Methinks this `.row()` is not necessary, try without it and see.
```suggestion
        add(titleTable).growX();
```